BuzzerWolf.Server/BuzzerWolf.Server.csproj

```diff
@@ -13,6 +13,7 @@
 			<PrivateAssets>all</PrivateAssets>
 			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
 		</PackageReference>
+		<PackageReference Include="NextjsStaticHosting.AspNetCore" Version="0.9.3-preview" />
```

Comment:
I installed this with package manager thru nuget, if setup instructions are required and not readily obvious let me know